pocketmine-mp/install.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+echo "Installing prerequisites.."
+
+sudo apt-get --yes --force-yes update -y
+sudo apt-get --yes --force-yes install \
+    ca-certificates \
+    curl \
+    gnupg \
+    lsb-release
+
+sudo mkdir -m 0755 -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+
+echo "Installing Docker Engine"
+
+sudo apt-get --yes --force-yes update
+sudo apt-get --yes --force-yes install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
```

Comment:
what would happen if the OS already have `docker` installed? I remember some images come with an extremely old or incompatible version of `docker`.

Reply:
If it already has docker it'll just say that it already has docker and skip that step

Reply:
I remember the official docker docs said something about uninstalling the default install first